examples/cra-kitchen-sink/.storybook/manager.js

```diff
@@ -9,6 +9,7 @@ addons.setConfig({
     base: 'light',
     brandTitle: 'CRA Kitchen Sink',
     brandUrl: 'https://github.com/storybookjs/storybook/tree/master/examples/cra-kitchen-sink',
+    brandTarget: '_self',
```

Comment:
We don't want to change our examples from default behavior, so this change should be removed please.

Reply:
Removed unnecessary changes